packages/tc-markdown-parser/lib/tree-mutators/coerce-treecreeper-properties-to-type.js

```diff
@@ -78,7 +78,13 @@ const coerceEnumPropertyValue = (
 	{ propertyType: enumName, hasMany, enums },
 ) => {
 	let [subdocument] = node.children;
-
+	const validValues = Object.values(enums[enumName]);
+	if (!subdocument.children.length) {
```

Comment:
just curious, is it not possible to have a validation as well (or do we have one) when the schema is added to `biz-ops-schema` than when it is coerced? 🤔 😕